server/core/src/main/java/org/infinispan/server/core/LifecycleCallbacks.java

```diff
@@ -66,7 +70,7 @@ private ConfigurationBuilder getServerStateCacheConfig(GlobalConfiguration globa
       return builder;
    }
 
-   Marshaller jbossMarshaller(ClassLoader classLoader, ClassWhiteList classWhiteList) {
+   public static Marshaller getJbossMarshaller(ClassLoader classLoader, ClassWhiteList classWhiteList) {
```

Comment:
I wonder if we should extract this to a different class, but I am not picky either way.

Reply:
That did cross my mind as well. Maybe we could add it to `org.infinispan.commons.util.Util` and then we can remove the logic from the client as well.